woocommerce/woocommerce
# Fix bug when filtering for customer_id=0.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes a small bug where empty results are returning when passing customer=0 as a param for REST API. This was happening because we were unsettling the `customer` param with an `if` statement, but it would also hit the if condition when param is set to `0`. This PR fixes it by changing the check to use `is_null` instead.

Closes #35852






### How to test the changes in this Pull Request:



1. Create an order with guest as customer (so that customer_id is 0).
2. Create another order with a logged in user as customer.
3. Send an API request with customer=0 param like so: `/wc/v3/orders?customer=0&_fields=id,customer_id&orderby=id&order=desc`. 
4. Check that guest order id is present, but logged customer id is not present.



### Other information:




### FOR PR REVIEWER ONLY:
